controller/lib/hal/hal_stm32.cpp

```diff
@@ -386,7 +387,6 @@ static void Timer15ISR() {
  * PA8  - Timer 1 Channel 1 - heater control
  * PA11 - Timer 1 Channel 4 - solenoid
  * PB3  - Timer 2 Channel 2 - blower control
- * PB4  - Timer 3 Channel 1 - buzzer
  *
  * For now I'll just set up the blower since that's the only
```

Comment:
I think this comment is no longer exactly true?

Reply:
I was referring to the statement that said only blower is being set up.

Reply:
Well, it's true that only the blower is being setup there.  I removed the reference to the buzzer since that moved to a different file.  I'm about to remove the reference to the solenoid since I'm starting work on the PSOL (also in it's own file)